src/bin/casr-cluster.rs

```diff
@@ -24,6 +24,31 @@ use std::process::Command;
 use std::process::Stdio;
 use std::sync::RwLock;
 
+/// Check whether some serde_json key array value is not null or empty
+///
+/// # Arguments
+///
+/// * `json` - serde_json
+///
+/// * `key` - array value key
+///
+/// * `path` - json file path
+///
+/// # Return value
+///
+/// bool
+fn is_not_empty_as_array(json: &serde_json::Value, key: &str, path: &Path) -> Result<bool> {
```

Comment:
It is a strange thing to use  Result for bool return type.

Reply:
Fixed